doc/Language/numerics.pod6

```diff
@@ -138,7 +138,7 @@ how-is-it 3+2i;  # OUTPUT: «meh␤»
 The types that do the L<Rational> role offer high-precision and
 arbitrary-precision decimal numbers. Since the higher the precision the
 larger the performance penalty, the L<Rational> types come in two flavours:
-L<Rat> and L<FatRat>. The L<Rat> is the most oft-used variant
+L<Rat> and L<FatRat>. The L<Rat> is the most often-used variant
```

Comment:
Actually, "oft" is OK. If you use often, you shouldn't use the dash.

Reply:
Oh, I wasn't aware of that. Thanks for pointing it out!